thumbor/context.py

```diff
@@ -67,11 +35,16 @@ def __init__(self, server=None, config=None, importer=None, request_handler=None
         self.config = config
         if importer:
             self.modules = ContextImporter(self, importer)
+            if importer.metrics:
+                self.metrics = importer.metrics(config)
+            else:
+                self.metrics = Metrics(config)
         else:
             self.modules = None
+            self.metrics = Metrics(config)
+
         self.filters_factory = FiltersFactory(self.modules.filters if self.modules else [])
         self.request_handler = request_handler
-        self.statsd_client = ThumborStatsClient.instance(config)
```

Comment:
this breaks backward compatibility with 3rd-party plugins so i'd either bump at least a minor version or provide some backward compatible version ie by aliasing the metrics variable.